libs/covid/data-api/src/lib/modules/counties/counties.service.ts

```diff
@@ -64,6 +64,20 @@ export class CountiesService extends BaseService<County> {
       t.lockdowns = curr.claims.reduce((a, c) => {
         return a + c.lockdowns.length;
       }, 0);
+      t.lockdownInfo = curr.claims.map((c) => {
+        if (c.lockdowns[0] === undefined) {
+          return null;
+        } else {
+          const lockdownStat: LockdownStat = {
+            updated: c.lockdowns[0].updated,
+
+            created: c.lockdowns[0].created,
+
+            guid: c.lockdowns[0].guid
```

Comment:
Remove spacing. 

Per our conversation, newlines should be placed between the `t` property assignments or before/after any thing beyond primitive value assignments (e.g. functions, objects, arrays), not between a list of any kind.

```js
      t.claims = { ... };

      t.sites = { ... };
      
      t.lockdowns = { ... };

      t.lockdownInfo = { ... };
```

Instead of:


```js
      t.claims = { ... };
      t.sites = { ... };   
      t.lockdowns = { ... };
      t.lockdownInfo = { ... };
```

Reply:
forgot to add new lines on the county stat object assignments. only did that for the lockdown info assignments at first. fixed it now.